components/northflank/actions/get-domains/get-domains.mjs

```diff
@@ -0,0 +1,33 @@
+import app from "../../northflank.app.mjs";
+
+export default {
+  key: "northflank-get-domains",
+  name: "Get Domains",
+  description: "List all domains with pagination. [See the documentation](https://northflank.com/docs/v1/api/domains/list-domains)",
+  version: "0.0.16",
+  type: "action",
+  props: {
+    app,
+    perPage: {
+      propDefinition: [
+        app,
+        "perPage",
+      ],
+    },
+    page: {
+      propDefinition: [
+        app,
+        "page",
+      ],
+    },
```

Comment:
Props `perPage` and `page` shouldn't be here. You need to use pagination to loop through and list all domains.
```suggestion
```